manifests/e/EclipseAdoptium/Temurin/8/8.0.332.9/EclipseAdoptium.Temurin.8.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.8
+PackageVersion: 8.0.332.9
+PackageLocale: en-US
+Publisher: Eclipse Foundation

```

Comment:
```suggestion
Publisher: Eclipse Adoptium
```

Reply:
This was updated using wingetcreate, so this mistake is present in all previous versions on WinGet packages. I'll correct this here but should I correct those versions too or correct them?